src/pykeen/nn/node_piece/anchor_selection.py

```diff
@@ -191,7 +192,7 @@ def extra_repr(self) -> Iterable[str]:  # noqa: D102
     # docstr-coverage: inherited
     def rank(self, edge_index: numpy.ndarray) -> numpy.ndarray:  # noqa: D102
         # sort by decreasing page rank
-        return numpy.argsort(page_rank(edge_index=edge_index, **self.kwargs))[::-1]
+        return numpy.argsort(page_rank(edge_index=torch.as_tensor(edge_index), **self.kwargs).numpy())[::-1]
```

Comment:
would we need `page_rank(...).detach().numpy()` or the computation would be always on cpu?

Reply:
d2ed47010d80dbf4b273a35e4a9b3e00af1f1122...00b0105d85ae2703e926e78666e3e95f61796f82